### PR TITLE
Bug 1988426: Change to use mountPath: /host

### PR DESCRIPTION
**Pull request by mccv1r0**
Change iptables chain names to not conflict with existing workaround(s)

Signed-off-by: Michael Cambria <mcambria@redhat.com>
(cherry picked from commit 32d72e5794ecd5e5f742c8708a2e4a6c183b0e4b)

### CODE REVIEW

File: bindata/network/openshift-sdn/sdn.yaml [hunk 1]
```diff
@@ -250,33 +250,33 @@ spec:
...
+          iptables -X AZURE_CHECK_ICMP_SOURCE || true
+          iptables -N AZURE_CHECK_ICMP_SOURCE || true
+          iptables -F AZURE_CHECK_ICMP_SOURCE
+          iptables -D INPUT -p icmp --icmp-type fragmentation-needed -j AZURE_CHECK_ICMP_SOURCE || true
+          iptables -I INPUT -p icmp --icmp-type fragmentation-needed -j AZURE_CHECK_ICMP_SOURCE
+          iptables -N AZURE_ICMP_ACTION || true
+          iptables -F AZURE_ICMP_ACTION
+          iptables -A AZURE_ICMP_ACTION -j LOG
+          iptables -A AZURE_ICMP_ACTION -j DROP
+          /host/usr/bin/oc observe pods -n openshift-sdn -l app=sdn -a '{ .status.hostIP }' -- /var/run/add_iptables.sh
```
Comment (squeed) [inline bindata/network/openshift-sdn/sdn.yaml:271]: No need to run `oc` here, just get the pod IP from the [downward api](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/).

(The pod is host-network anyways).

Comment (squeed) [reply bindata/network/openshift-sdn/sdn.yaml:271]: argh, didn't realize this was a backport. The usage of `oc` should be removed here at some point.

**Comment by squeed** [comment]
Do we need to do anything to clean up existing nodes? Remember, this will be applied unrelated to reboots.

File: bindata/network/openshift-sdn/sdn.yaml (hunk 1 above)
Comment (mccv1r0) [reply bindata/network/openshift-sdn/sdn.yaml:271]: We need the hostIP of all the other nodes in the cluster, as they are added.

**Comment by deads2k** [comment]
(patch manager)

urgent bz, 1.5

**Pull request merged**